src/plugins/dashboard/public/dashboard_actions/filters_notification_action.tsx

```diff
@@ -103,9 +96,7 @@ export class FiltersNotificationAction implements Action<EmbeddableApiContext> {
   ) {
     if (!isApiCompatible(embeddable)) return;
     return merge(
-      ...[embeddable.query$, embeddable.filters$, getViewModeSubject(embeddable)].filter((value) =>
-        Boolean(value)
-      )
+      ...[embeddable.query$, embeddable.filters$].filter((value) => Boolean(value))
```

Comment:
That's true, but since we likely will want to edit panel level filters from the inline editor in the near future as part of the upcoming inline edit effort, I'd say we just leave this logic in for now instead of deleting it.